setup_scripts/99-phidgets.rules

```diff
@@ -0,0 +1,7 @@
+# Very old Phidgets
+SUBSYSTEMS=="usb", ACTION=="add", ATTRS{idVendor}=="0925", ATTRS{idProduct}=="8101", MODE="666"
+SUBSYSTEMS=="usb", ACTION=="add", ATTRS{idVendor}=="0925", ATTRS{idProduct}=="8104", MODE="666"
+SUBSYSTEMS=="usb", ACTION=="add", ATTRS{idVendor}=="0925", ATTRS{idProduct}=="8201", MODE="666"
+
+# All current and future Phidgets - Vendor = 0x06c2, Product = 0x0030 - 0x00af
+SUBSYSTEMS=="usb", ACTION=="add", ATTRS{idVendor}=="06c2", ATTRS{idProduct}=="00[3-a][0-f]", MODE="666"
```

Comment:
Did you confirm that apt does/does not setup udev rules? I'd rather we copy from the system phidget folder (under `share/ros`) if it doesn't, just because this file will likely become quickly out of date if the `phidget_api` package changes.

Reply:
Agree, added `phidgets_drivers` as a dependency for `localisation_igvc` and takes the udev rules from the installed package at `/opt/ros/kinetic/share/phidgts_api/udev`.